src/unit/test_files.h

```diff
@@ -23,6 +23,18 @@ int test_kvstoreIteratorRemoveAllKeysNoDeleteEmptyDict(int argc, char **argv, in
 int test_kvstoreIteratorRemoveAllKeysDeleteEmptyDict(int argc, char **argv, int flags);
 int test_kvstoreDictIteratorRemoveAllKeysNoDeleteEmptyDict(int argc, char **argv, int flags);
 int test_kvstoreDictIteratorRemoveAllKeysDeleteEmptyDict(int argc, char **argv, int flags);
+int test_randomWalkTest(int argc, char **argv, int flags);
```

Comment:
(Nit) In this file, all unit tests from all c files are mixed so it's not easy to see which ones are rax tests. Some names like "test_fuzz" and "test_benchmark" are a bit too generic too. But I don't want to complain too much since this problem isn't in this PR. If you want, you can add some comment before this group of tests and a blank line after the group...

```suggestion
/* rax tests */
int test_randomWalkTest(int argc, char **argv, int flags);
```

And a blank line after this group.